cli/src/main/scala/com/lightbend/rp/reactivecli/Main.scala

```diff
@@ -16,17 +16,47 @@
 
 package com.lightbend.rp.reactivecli
 
-import libhttpsimple.LibHttpSimple
+import libhttpsimple.{ LibHttpSimple, HttpRequest }
 import scopt.OptionParser
+import slogging._
 import argonaut._
 import Argonaut._
 
-object Main {
-  case class InputArgs(foo: Option[String] = None)
+object Main extends LazyLogging {
+  object LogLevels extends Enumeration {
+    type Level = Value
+
+    val error, warn, info, debug, trace = Value
+  }
+
+  implicit val logLevelsRead: scopt.Read[LogLevels.Value] =
+    scopt.Read.reads(LogLevels.withName)
+
+  case class InputArgs(foo: Option[String] = None, logLevel: LogLevels.Value = LogLevels.info)
 
   val defaultInputArgs = InputArgs()
 
-  implicit def inputArgsCodecJson = casecodec1(InputArgs.apply, InputArgs.unapply)("foo")
+  implicit val logLevelsCodecJson = CodecJson[LogLevels.Value](
```

Comment:
If this done only to display the JSON on the main file, I'd suggest we remove it as I can definitely confirm the JSON parser works as expected from #10